### PR TITLE
Add @ExpectMaxInsert

### PR DESCRIPTION
**Pull request by MrSnix**

### CODE REVIEW

**Comment by MrSnix** [comment]
@jeanbisutti I have no clue why it is failing only on openjdk14 and oraclejdk9, seems like the cause is related to AbstractJUnit4SpringTestBase, particularly the following method    "a_test_method_having_a_performance_property_not_respected_in_a_dedicated_jvm_should_fail()",
is expecting zero, but got one.

**Comment by jeanbisutti** [comment]
@MrSnix Thank you for your PR. There is an issue with the Travis build, for several jobs the JDK installed is different from the JDK used. I created a Travis topic: https://travis-ci.community/t/the-jdk-used-is-different-from-the-jdk-installed/8649

**Comment by jeanbisutti** [comment]
@MrSnix The build problem is fixed now. I am going to review your PR in the next days.

**Comment by jeanbisutti** [comment]
Sorry for the delay. Many thanks for this contribution!

**Comment by jeanbisutti** [comment]
Could you please add a builder method in `JvmAnnotationBuilder` class ?

**Comment by jeanbisutti** [comment]
Many thanks @MrSnix for this PR!

The feature will be great with some improvements on user suggestions!

**Comment by MrSnix** [comment]
I'll add the builder method but cannot do it right now, you'll have to wait a few day.
I'm sorry.

**Comment by jeanbisutti** [comment]
> I'll add the builder method but cannot do it right now, you'll have to wait a few day.
I'm sorry.

No problem, when it will be possible for you.

Many thanks for the time you spend working on QuickPerf. :pray:

**Comment by MrSnix** [comment]
I have some trouble understanding which builder method i have to add inside the JvmAnnotationBuilder class if ExpectMaxInsert is an interface inside the sql-annotation package...
Why it's a jvm annotation? What i have to do? Could you provide me with an example?

**Comment by jeanbisutti** [comment]
I made a mistake. Sorry about that.

A builder method has to be added into **Sql**AnnotationBuilder.

File: sql/sql-annotations/src/main/java/org/quickperf/sql/insert/MaxOfInsertsPerfIssueVerifier.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
+package org.quickperf.sql.insert;
```
[the remark this answers is not in the document]
Comment (MrSnix) [reply sql/sql-annotations/src/main/java/org/quickperf/sql/insert/MaxOfInsertsPerfIssueVerifier.java:1]: Done

**Comment by MrSnix** [comment]
> I made a mistake. Sorry about that. A builder method has to be added into **Sql**AnnotationBuilder:
> 
> ```java
>    public static ExpectMaxInsert expectMaxInsert(final int value) {
>        return new ExpectMaxInsert() {
>            @Override
>            public Class<? extends Annotation> annotationType() {
>                return ExpectMaxInsert.class;
>            }
>            @Override
>            public int value() {
>                return value;
>            }
>        };
>    }
> ```

Did you check that? I have already done it.

Btw, i should have implemented all the changes and suggestions you gave to me.
Please, check if i have missed something.
Thanks

**Comment by jeanbisutti** [comment]
It's great!
I made a [PR](https://github.com/MrSnix/quickperf/pull/1) for your Github repo with some other improvements.
Could you please merge this PR?
I am going to release QuickPerf shortly. I propose to merge your PR just after this release. The release containing your PR will contains several features linked to JDBC batching.
Many thanks for your work.

**Comment by MrSnix** [comment]
I have correctly merged your pull request

**Comment by jeanbisutti** [comment]
@MrSnix [The release is done](https://github.com/quick-perf/quickperf/releases/tag/release-1.0.0-RC7). Could you please rebase on master and squash the commits into one? After that I could merge the PR.

**Comment by jeanbisutti** [comment]
I see several merges in the Git history.

We try to get a clean Git history.

One last thing, could you please create a new PR?

Sorry about that...

To make the thing easier, I listed all the files that have to be in the commit:
* sql/sql-annotations/src/main/java/org/quickperf/sql/annotation/ExpectMaxInsert.java 
* sql/sql-annotations/src/main/java/org/quickperf/sql/annotation/SqlAnnotationBuilder.java 
* sql-annotations/src/main/java/org/quickperf/sql/config/library/SqlAnnotationsConfigs.java 
* sql/sql-annotations/src/main/java/org/quickperf/sql/config/library/SqlConfigLoader.java 
* sql/sql-annotations/src/main/java/org/quickperf/sql/framework/HibernateSuggestion.java 
* sql/sql-annotations/src/main/java/org/quickperf/sql/framework/SpringDataJpaSpringBootSuggestion.java
* sql/sql-annotations/src/main/java/org/quickperf/sql/insert/InsertNumberPerfIssueVerifier.java
* sql/sql-annotations/src/main/java/org/quickperf/sql/insert/MaxOfInsertsPerfIssueVerifier.java
* sql/sql-integration-test/src/test/java/ExpectMaxInsertTest.java

Many thanks for your work.

**Pull request closed without merging**